src/gsy_e/models/strategy/load_hours.py

```diff
@@ -50,12 +50,154 @@
 BalancingRatio = namedtuple("BalancingRatio", ("demand", "supply"))
 
 
+class LoadHoursEnergyParameters:
```

Comment:
Nit: I'd go even further and create a module structure. For example:
```
some-parent-folder
├── parameters
│   ├── load_hours_parameters.py
│   └── pv_parameters.py
└── strategies
    ├── full_gsye  # I'm not sure about naming
    │   ├── load_hours_strategy.py
    │   └── pv_strategy.py
    └── scm
        ├── load_hours_strategy.py
        └── pv_strategy.py
```
What do you think? (Just a nit anyway, no need to do anything here)

Reply:
I agree, but I will create a followup PR for this in order to not further complicate this one, thanks!